src/Medidata.ZipkinTracer.Owin/ZipkinTracerOwinExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Web;
+using Microsoft.Owin;
+
+namespace Medidata.ZipkinTracer.Owin
+{
+    internal static class ZipkinTracerOwinExtensions
+    {
+        public static readonly string HttpContextBaseKey = "System.Web.HttpContextBase";
```

Comment:
No magic string is better?
`public static readonly string HttpContextBaseKey = typeof(HttpContextBase).FullName;`